main/monmon.rb

```diff
@@ -1,14 +1,15 @@
 require 'csv'
 
-balance = Hash.new
+balance = Hash.new(0)
 
-CSV.foreach("input.csv", headers: true) do |row|
-  puts("#{row['Type']}: #{row['Name']}:  #{row['Currency']}\t - #{row['Amount']}  #{row['Currency']}")
-  if balance.include?(row['Currency'])
-    balance[row['Currency']] += row['Amount'].to_i
+CSV.foreach("input.csv", :headers => true, :header_converters => :symbol) do |row|
+  puts("#{row[:type]}: #{row[:name]}:  #{row[:currency]}\t - #{row[:amount]}  #{row[:currency]}")
+  if balance.include?(row[:currency].to_sym)
```

Comment:
you don't need this if anymore since hash has default value = 0.

Reply:
I agree